assembl/graphql/schema.py

```diff
@@ -133,6 +135,24 @@ class Query(graphene.ObjectType):
         end_date=graphene.String(description=docs.SchemaPosts.end_date),
         identifiers=graphene.List(graphene.String, description=docs.SchemaPosts.identifiers),
         description=docs.SchemaPosts.__doc__)
+    tags = graphene.List(
+        lambda: Tag,
+        filter=graphene.String(description=docs.SchemaTags.filter),
+        description=docs.SchemaTags.__doc__)
+
+    def resolve_tags(self, args, context, info):
+        discussion_id = context.matchdict['discussion_id']
+        _filter = args.get('filter', '')
+        model = models.Keyword
+        query = get_query(model, context).filter(
+            model.discussion_id == discussion_id)
+
+        # @TODO Add Pagination
```

Comment:
If you want pagination, transform List to a SQLAlchemyConnectionField and remove the `.all()` in the query below.